test-amp-list.js

```diff
@@ -0,0 +1,417 @@
+/**
```

Comment:
Remove this file?

Reply:
Oh weird! Something strange must have happened during rebase. 0.0